src/basilisp/core.lpy

```diff
@@ -829,6 +833,11 @@
   [& args]
   (not (apply = args)))
 
+(defmacro ==
+  "Alias for lpy:fn:`=`."
+  [& args]
+  `(= ~@args))
```

Comment:
I think this being a macro could be surprising. One for instance could not `(map == [1 2 3])` with it. I think since it's probably relatively infrequently utilized (I personally have never seen it in the wild) it can simply be:

```clojure
(defn ==
  "Alias for lpy:fn:`=`."
  [& args]
  (apply = args))
```

Reply:
Right, corrected,  it was just me not thinking, thanks